charts/cass-operator-chart/Chart.yaml

```diff
@@ -2,3 +2,9 @@ apiVersion: v2
 name: cass-operator
 version: 1.4.0
 description: Helm chart for Cass Operator.
+appVersion: 1.4.0
+home: https://github.com/datastax/cass-operator
+maintainers:
+  - name: Cassandra Operator Team
+    email: cass-operator@datastax.coom
```

Comment:
typo: coom

Reply:
Oof. I'll get this changed in a follow-up. This may require a change in the charts repo too.